pkg/injector/pod_patch_test.go

```diff
@@ -113,3 +115,101 @@ func TestGetSideCarContainer(t *testing.T) {
 	assert.Equal(t, "secret", container.Env[2].ValueFrom.SecretKeyRef.Name)
 	assert.EqualValues(t, expectedArgs, container.Args)
 }
+
+func TestSetDaprEnvVars(t *testing.T) {
+	mockContainer := corev1.Container{}
```

Comment:
Please refactor this test code by using [subtests](https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks) and [TableDriventests](https://github.com/golang/go/wiki/TableDrivenTests#example-of-a-table-driven-test).